lib/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "gatsby-source-storyblok",
+  "contributors": [
+    {
+      "name": "Alexander Feiglstorfer <delooks@gmail.com>"
+    }
+  ],
+  "version": "4.0.0",
+  "description": "SDK to integrate Storyblok into your project using Gatsby.",
+  "main": "./dist/gatsby-source-storyblok.js",
+  "module": "./dist/gatsby-source-storyblok.mjs",
+  "files": [
+    "gatsby-node.js",
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "dev": "vite build --watch",
+    "build": "vite build",
+    "test": "npm run test:unit && npm run test:e2e",
+    "test:unit": "jest __tests__",
+    "test:e2e": "start-server-and-test cy:playground http-get://localhost:8000/ cy:run",
+    "test:e2e-watch": "start-server-and-test cy:playground http-get://localhost:8000/ cy:open",
+    "cy:playground": "npm run develop --prefix ../playground-test",
+    "cy:run": "cypress run",
+    "cy:open": "cypress open",
+    "prepublishOnly": "npm run build && cp ../README.md ./"
+  },
+  "dependencies": {
+    "@storyblok/react": "^1.0.5",
+    "storyblok-js-client": "^4.5.2"
```

Comment:
I think we don't need this dependency. The JS client is already included in `@storyblok/js` and, because of that, it's also included in `@storyblok/react`

Reply:
I believe we still need @storyblok/react in dependency as Gatsby SDK needs to import the JS Client and other APIs coming from JS and React SDKs. What do you think?

Reply:
I think Facundo means storyblok-js-client. Normally that'd be true, but in this case it's used by `gatsby-node.js` which is a node module and we'd need a bit of refactoring to remove the dep

Reply:
Yes, as @alexjoverm says I was talking about `storyblok-js-client`.

Thanks for the clarification, I didn't see it being used in `gatsby-node.js` before